ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -868,14 +848,28 @@ func (e *ExecutableHandlerImpl) SetupAccApfs() error {
 	return nil
 }
 
-func (s *FXPHandlerImpl) configureFXP(p4rtbin string, brCtlr types.BridgeController) error {
-	vfMacList, err := utils.GetVfMacList()
-	if err != nil {
-		return fmt.Errorf("Unable to reach the IMC %v", err)
-	}
-	if len(vfMacList) == 0 {
-		return fmt.Errorf("No NFs initialized on the host")
+// If ipu-plugin's Init function gets invoked on ACC, prior to getting invoked
+// on x86, then host VFs will not be setup yet. In that case, peer2peer rules
+// can get added in CreateBridgePort or CreateNetworkFunction.
```

Comment:
Not sure I understood. "Can get added" or "will get added"? Does the dpu daemon have to send those or will the ipu plugin have intelligence to automatically program those?

Reply:
Is it added there today?